packages/AdaptiveCards/Schemas/Microsoft.Bot.Components.GetAdaptiveCardTemplate.schema

```diff
@@ -0,0 +1,45 @@
+﻿{
+  "$schema": "https://schemas.botframework.com/schemas/component/v1.0/component.schema",
+  "$role": "implements(Microsoft.IDialog)",
+  "title": "Get an Adaptive Card Template",
+  "description": "Loads and optional caches an Adaptive Card Template from a Url.",
```

Comment:
optionally

Reply:
good catch

---
In reply to: [610231258](https://github.com/microsoft/botframework-components/pull/862#discussion_r610231258) [](ancestors = 610231258)